README.markdown

```diff
@@ -1476,6 +1477,27 @@ The compiler will show a warning ---> 'titleLabel' is deprecated: This has to be
     }
 ```
 
+## Objc annotation
+
+Sometimes we need to add an `@objc` annotation to a `var`, `let` or `func`. In that case we'll leave the `@objc` annotation in its own line, add a line break and then the statement it's annotating in a new line.
+
+**Preferred**:
+
+```swift
+@objc 
+func refreshAllProjects(deleteAllData: Bool = true) {
+        retrieveProjects()
```

Comment:
whitespace :P

Reply:
Fixed in 9f1e2ef8163b303f4813d5d3c8eb866c05a44fe0